Purchases/Identity/IdentityManager.swift

```diff
@@ -61,9 +65,9 @@ class IdentityManager {
     }
 
     func logIn(appUserID: String, completion: @escaping (CustomerInfo?, Bool, Error?) -> Void) {
-        let newAppUserID = appUserID.trimmingCharacters(in: .whitespacesAndNewlines)
+        let newAppUserID = appUserID.trimmingWhitespacesAndNewLines
         guard !newAppUserID.isEmpty else {
-            Logger.error(Strings.identity.logging_in_with_nil_appuserid)
```

Comment:
This method doesn't allow `nil`, so the old message was slightly incorrect.
Also, using `nil` is explicitly allowed and documented to be a valid input.

Reply:
probably inherited from objc days